src/app/shared/view-mode-switch/view-mode-switch.component.html

```diff
@@ -6,7 +6,7 @@
        routerLinkActive="active"
        [class.active]="currentMode === viewModeEnum.List"
        class="btn btn-secondary">
-          <i class="fas fa-list" title="{{'search.view-switch.show-list' | translate}}"></i>
+          <i class="fa fa-list" title="{{'Show as list' | translate}}"></i>
```

Comment:
Also a change from `fas` to `fa`.

Reply:
Unclear how that replace happened, reverted today.